src/sbm.jl

```diff
@@ -78,6 +78,12 @@
     temperature::Vector{T} | "°C"
     # Potential evapotranspiration [mm Δt⁻¹]
     potential_evaporation::Vector{T}
+    # Multiplication factor [-] for epot
+    epot_factor::Vector{T} | "-"
```

Comment:
This is somewhat double on top of the `et_reftopot` factor, though they do serve different purposes, so I guess it's fine to have both.

Reply:
yes, also noticed this, but agree with you that these have different purposes, so I would also leave it like this